examples/blueprint-construct/index.ts

```diff
@@ -5,7 +5,7 @@ import { PolicyStatement } from 'aws-cdk-lib/aws-iam';
 import { Construct } from "constructs";
 import * as blueprints from '../../lib';
 import { AckServiceName, HelmAddOn } from '../../lib';
-import { EmrEksTeamProps } from '../../lib/teams';
+import { EmrEksTeamProps, BatchEksTeamProps, BatchAllocationStrategy, BatchEnvType } from '../../lib/teams';
```

Comment:
@youngjeong46 we use wildcard import if more than 3 objects are imported. This seems redundant since we `import * as teams` on line 10. You should be able to use `teams.BatchEksTeamProps` and such.

Reply:
the import on line 10 is from the `../teams` as in the custom teams from examples. This import is from our standard library of teams. Regardless your comment stands and I will use wildcards.